README.md

```diff
@@ -76,6 +76,11 @@ There are some areas in which there is no clear consensus in datarockets team re
 In such scenarios, all popular styles are acknowledged and it’s up to you to pick one and apply it consistently.
 For that just set up these cops before starting of usage.
 
+#### RSpec/LetSetup
+
+[This cop](https://rubocop-rspec.readthedocs.io/en/latest/cops_rspec/#rspecletsetup) is enabled by default and we suggest not to use unreferenced `let` variables in your test cases.
+But you feel that unreferenced `let` variables are needed and test can't be written w/p them, please create an issue with your case so that we can re-thinking our solution about enabling this cop.
```

Comment:
> But you feel that unreferenced `let` variables are needed

To

> However, if it feels like the cop should be enabled

?

Reply:
`be written w/p them` -> `be written w/o them`

Reply:
`and test can't be` -> `and a test can't be`

Reply:
👍

Reply:
P.S. `and test can't be` -> `and tests can't be`